src/client/utils/url.js

```diff
@@ -86,7 +87,7 @@ export function changeDestUrlPart (proxyUrl, prop, value, resourceType) {
     var parsed = sharedUrlUtils.parseProxyUrl(proxyUrl);
 
     if (parsed) {
-        var resolver  = destLocation.getResolver(document);
+        var resolver  = urlResolver.getResolverElement(document);
```

Comment:
I would have moved all associated with the resolver element code in the urlResolver